scripts/config/Makefile

```
@@ -33,11 +37,22 @@ clean-files	:= zconf.tab.c zconf.lex.c zconf.hash.c
 # Remove qconf junk files
 clean-files	+= $(qconf-cxxobjs) qconf.moc .tmp_qtcheck qconf
 
-all: conf mconf
+all: conf mconf nconf
```

Not sure it's correct, but I did not understand how `qconf` is compiled for `make xconfig`.

---

```suggestion
all: conf mconf
```
You shouldn't add it to `all`, otherwise you'll risk not being able to build `mconf` and `menuconfig` because of it.  Change `make all` to `make nconf` as I'm mentioning above, and it will only build `nconf`.